Thermal_Expansion/assets/thermalexpansion/textures/blocks/Issues.txt

```diff
@@ -0,0 +1,6 @@
+Issues within T.E. block textures:
```

Comment:
These files are not allowed in the repo layout. The only kind of `.txt` file allowed is an `~Untextured.txt` which goes in each folder of what textures are not _"textured"_. But they are not **required** as when mods are updated or support is added for a texture we just commit them in. 

Along with the Untextured files get out of date really quickly. So we don't really use them at all. But these Issues files would need removing before regardless 🙂